pkg/graph/schema.graphql

```diff
@@ -100,6 +100,7 @@ type CedarSystem {
 	status: String
 	businessOwnerOrg: String
 	businessOwnerOrgComp: String
+  businessOwnerRoles: [CedarRole!]!
```

Comment:
nit: indentation

Reply:
This doesn't seem to show up when viewing the [file](https://github.com/CMSgov/easi-app/blob/9bde3670cb16cb7c4a02dd71b4df506166ace7e5/pkg/graph/schema.graphql#L95-L107).

Must be a GitHub formatting difference between tabs/spaces.